dev-libs/libpeas/metadata.xml

```diff
@@ -6,6 +6,7 @@
     <name>Gentoo GNOME Desktop</name>
   </maintainer>
   <use>
+    <flag name="gjs">Enable GJS support using <pkg>dev-libs/gjs</pkg></flag>
```

Comment:
I think we could use the global "javascript" name. It seems upstream pretends to support javascript (via GJS)
https://gitlab.gnome.org/GNOME/libpeas/-/issues/47

Reply:
Done in https://github.com/gentoo/gentoo/pull/34835/commits/6cddcffa77b0823b7a569da486e87228203288b7